stable/proxysql/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.enable }}
```

Comment:
I dont think the enabled check should be added here... it should be added when adding it as dependency

Reply:
I don't get it. Could you explain?

Reply:
I mean these checks should be removed from here, and when adding this chart as a dependency we can have a conditional check like
```
dependencies:
  - name: subchart1
    condition: subchart1.enabled
```
this way this chart wont even be downloaded into the dependencies

Reply:
`{{- if .Values.enable }}` removed